_source/_posts/2023-01-23-keycloak-kubernetes-prod.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: blog_post
+title: "OAuth2 microservices with Keycloak on Google Kubernetes Engine"
```

Comment:
I just copied the keywords for this post into the PR as a comment. We're trying to capture `spring boot kubernetes` as keywords, but I realize that might be tough. Here's some suggestions for titles with their scores:

- Keycloak and Spring Boot in Production: 58
- Build a Spring Boot App with Keycloak and Kubernetes: 57
- Keycloak in Production with Kubernetes: 56
- Keycloak in Production with Spring Boot and JHipster: 61

We should also try to use these keywords in the first few headings for SEO.

Reply:
Added the keywords in the title and headings.